packages/node-core/src/configure/configure.module.spec.ts

```diff
@@ -93,7 +93,9 @@ describe('Configure', () => {
       const option = {headers: {'api-key': '<your-api-key>'}};
       const {primaryNetworkEndpoint} = yargsToIConfig({
         'primary-network-endpoint': 'https://example.com',
+        primaryNetworkEndpoint: 'https://example.com',
```

Comment:
I don't think this is a sufficient test. The issue was the snake case being overwritten not support for snakeCase

Reply:
The fields we get from yargs will include both snake_case and camelCase.